client/blocks/comment-detail/index.jsx

```diff
@@ -239,9 +240,12 @@ export class CommentDetail extends Component {
 			isExpanded,
 		} = this.state;
 
+		const commentIsPing = 'pingback' === commentType || 'trackback' === commentType;
```

Comment:
Nitpick: I don't find this very readable and it will be even less so when we add `'reactions'` here. :smile: We could replace it with `_.includes`.